phoebe/atmospheres/passbands.py

```diff
@@ -643,6 +479,15 @@ def load(cls, archive, load_content=True, init_extrapolation=True):
 
         logger.debug("loading passband from {}".format(archive))
 
+        atms = {
+            'ck2004': 'CK',
+            'phoenix': 'PH',
+            'tmap_sdO': 'TS',
+            'tmap_DA': 'TA',
+            'tmap_DAO': 'TM',
+            'tmap_DO': 'TO'
+        }
```

Comment:
this is in multiple places, can we put it somewhere common and import it?

Reply:
yep, done; will be reflected in the next commit.